### PR TITLE
feat: add yarn 1 (classic) examples

### PR DESCRIPTION
**Pull request by MikeMcC399**
This PR is related to issue https://github.com/cypress-io/github-action/issues/799 "Examples for Yarn 1 (Classic) and Yarn Berry / Modern (Version 2 and later)".

It provides new examples for Yarn Classic under the directories:

- `examples/yarn-classic`
- `examples/v9/yarn-classic`

together with a new corresponding workflow

- `example-yarn-classic.yml`

an update to the following script to automate maintenance updates to `cypress@latest`

- `scripts/update-cypress-latest-other.sh`

an update to cover both Yarn Classic (v1) and Yarn Modern (V2 and later) in the following file

- `.gitignore`

 and an update to

- `README.md`

### Local

```bash
nvm use 18 # for node v18.15.0
npm install yarn -g
yarn -v
```

expect `1.22.19`

#### Yarn Classic with Cypress 9.7.0

From root of repository ...

```bash
cd examples/v9/yarn-classic
yarn
yarn test
```

Confirm Cypress test passes.

#### Yarn Classic with Cypress 12.7.0

From root of repository ...

```bash
cd examples/yarn-classic
yarn
yarn test
```

### Update script

From root of repository ...

```bash
cd examples/yarn-classic
yarn add cypress@12.0.0 -D # install older version
cd ../..
npm run update:cypress
cd examples/yarn-classic
yarn list --pattern cypress
```

Confirm that Cypress has been updated to latest version (12.7.0).

### GitHub

Manually run the workflow

- `example-yarn-classic.yml`

and confirm success.

### CODE REVIEW

**Comment by MikeMcC399** [comment]
All manual tests pass.

I will follow up this PR with a Yarn Modern example PR. It is already prepared and needs this Yarn 1 (Classic) PR merging first.

File: scripts/update-cypress-latest-other.sh [hunk 1]
```diff
@@ -33,7 +33,15 @@ cd install-command
...
 
+# examples/yarn-classic
+echo
+echo updating examples/yarn-classic to cypress@latest
+cd yarn-classic
+yarn add cypress --dev --exact
+cd ..
+
 # examples/basic-pnpm (pnpm)
+echo
```
Comment (AtofStryker) [inline scripts/update-cypress-latest-other.sh:44]: is this just to print a new line?

**Review by AtofStryker — APPROVED** [review]

**Comment by AtofStryker** [comment]
@lmiller1990 do you mind taking a look at this one?

File: scripts/update-cypress-latest-other.sh (hunk 1 above)
Comment (MikeMcC399) [reply scripts/update-cypress-latest-other.sh:44]: > is this just to print a new line?

Yes, that's right. It's just cosmetic.

**Review by lmiller1990 — APPROVED** [review]

**Comment by lmiller1990** [comment]
Thanks!

**Comment by MikeMcC399** [comment]
- PR #838 follows on from this PR

**Pull request merged**